examples/config_geturl.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
You can also update the documentation: https://asab.readthedocs.io/en/latest/asab/config.html

Reply:
Ok I'll look into it

Reply:
Docs updated :)